### PR TITLE
dist/Math-Complex - add missing files from CPAN dist

### PR DESCRIPTION
**Pull request by demerphq**
This fixes this module for https://github.com/Perl/perl5/issues/20874

### CODE REVIEW

File: dist/Math-Complex/Makefile.PL [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{ use 5.006; }
+
+use ExtUtils::MakeMaker;
+
+WriteMakefile(
+	      NAME            => 'Math::Complex',
+	      VERSION_FROM    => 'lib/Math/Complex.pm',
+	      INSTALLDIRS     => 'perl',
```
Comment (tonycoz) [inline dist/Math-Complex/Makefile.PL]: INSTALLDIRS should be conditional on perl version:
```
INSTALLDIRS => ( $] < 5.011 ? 'perl' : 'site' ),
```

Comment (tonycoz) [reply dist/Math-Complex/Makefile.PL]: Sorry for the duplication, but I thought a comment on each PR would make it easier to track.

**Review by tonycoz — APPROVED** [review]

File: dist/Math-Complex/Makefile.PL (hunk 1 above)
Comment (demerphq) [reply dist/Math-Complex/Makefile.PL]: Fixed.

Comment (demerphq) [reply dist/Math-Complex/Makefile.PL]: > Sorry for the duplication, but I thought a comment on each PR would make it easier to track.

Nothing to be sorry about, you were right. It did make it easier to track. :-)

**Pull request merged**